cookbook/logging/monolog.rst

```diff
@@ -26,6 +26,60 @@ The ``logger`` service has different methods for different logging levels.
 See :class:`Symfony\\Component\\HttpKernel\\Log\\LoggerInterface` for details
 on which methods are available.
 
+Since version `2.11 of MonologBundle <https://github.com/symfony/monolog-bundle/releases/tag/2.11.0>`
+an extra parameter for disabling the microsecond precision is available. This is
+kind of a micro optimization but could speed up log generation, specially if you log
+a lot of debug messages.
```

Comment:
can we maybe add a little explanation about what it is disabling and in which cases it should be enabled?